src/main_operations/modules/apriltags/pre_processing/ai_accelleration/training/augmentation/augment_data.py

```
@@ -0,0 +1,407 @@
+        # Check if corresponding JSON file exists
+        json_file = png_file.replace(PNG_EXTENSION, JSON_EXTENSION)
+        if not os.path.exists(json_file):
+            orphaned_count += 1
+
+    return orphaned_count
+
+
+def augment_training_data(
+    input_dir: str,
+    output_dir: str,
+    enabled_augmentations: List[str] = None,
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)